src/odemis/acq/stream/_projection.py

```diff
@@ -1013,10 +1013,15 @@ def _projectTile(self, tile):
         """
         dims = tile.metadata.get(model.MD_DIMS, "CTZYX"[-tile.ndim::])
         ci = dims.find("C")  # -1 if not found
-        # is RGB
-        if dims in ("CYX", "YXC") and tile.shape[ci] in (3, 4):
-            # Just pass the RGB data on
+        tint = tuple(self.stream.tint.value)
```

Comment:
Altough tint is a normal word, I (and Bassim) had problems understanding that it changed somthing from gray scale to a scale based on antother color. I keep thinking it is probably a 't'-type integer (t-int) so it would help to add behind this line the comment:
" value to change the tint/colorscale"

But again, tint is a normal English word.

Reply:
I understand the confusion with "int". That said, this file (and acq.stream._base) has many places with "tint". So I'd rather leave it like this here.

Reply:
Ok, I understand that.
Ready to Merge! (should I now press the button since I last reviewed or do you(@pieleric ) still do that?

Reply:
Usually, I take care of merging. Although everyone can do it.